Test.C

```diff
@@ -224,14 +265,18 @@ int main(){
                     // keep in bounds
                     if (encryptionStock[i].effectiveness < 0)
                         encryptionStock[i].effectiveness = 0;
-                    printf(" - %s effectiveness has been reduced to %.2f!\n", encryptionStock[i].name, encryptionStock[i].effectiveness);
+                    printf(" - %s effectiveness has been reduced to %.0f%!\n", encryptionStock[i].name, encryptionStock[i].effectiveness * 100);
```

Comment:
Need to escape `%` character.
```suggestion
                    printf(" - %s effectiveness has been reduced to %.0f%%!\n", encryptionStock[i].name, encryptionStock[i].effectiveness * 100);
```